src/query/ast/src/parser/query.rs

```diff
@@ -407,12 +429,29 @@ impl<'a, I: Iterator<Item = WithSpan<'a, TableReferenceElement<'a>>>> PrattParse
                 name,
                 params,
                 alias,
-            } => TableReference::TableFunction {
-                span: input.span.0,
-                name,
-                params,
-                alias,
-            },
+            } => {
+                let normal_params = params
+                    .iter()
+                    .filter_map(|p| match p {
+                        TableFunctionParam::Normal(p) => Some(p.clone()),
+                        _ => None,
+                    })
+                    .collect();
+                let named_params = params
+                    .into_iter()
+                    .filter_map(|p| match p {
+                        TableFunctionParam::Named { name, value } => Some((name, value)),
+                        _ => None,
+                    })
+                    .collect();
```

Comment:
Should we check that all named_params are behind normal params by `params.iter().array_windows::<2>().all(...)`

Reply:
This is unnecessary.